### DISCUSSION TITLE
Features in v4

### DISCUSSION THREAD

**Discussion by rudgej** [Blackymas/NSPanel_HA_Blueprint#1175] [General]
I am currently on version 3.x. 

I can't find any release notes for version 4 showing the new features. 

Can anyone point me to the release notes please ? Thank you. Jeremy

**Comment by edwardtfn** [comment]
Release notes:
- v4.0: https://github.com/Blackymas/NSPanel_HA_Blueprint/releases/tag/v4.0
- v4.0.1: https://github.com/Blackymas/NSPanel_HA_Blueprint/releases/tag/v4.0.1
- v4.0.2: https://github.com/Blackymas/NSPanel_HA_Blueprint/releases/tag/v4.0.2
- All release notes: https://github.com/Blackymas/NSPanel_HA_Blueprint/releases

**Reply by rudgej** [reply]
Thank you
